ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -193,7 +193,8 @@ tests:
     cluster_profile: packet-assisted
     env:
       POST_INSTALL_COMMANDS: |
-        export TEST_FUNC=test_kube_api_ipv4
+        export TEST_FUNC=test_kubeapi
```

Comment:
I think you can remove this test
Or change name schema to make it more clear what it does

Reply:
I'll remove it, I'm not sure what it does